go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -62,12 +62,20 @@ func TestTabletInitialBackup(t *testing.T) {
 	// Initialize the tablets
 	initTablets(t, false, false)
 
-	// Restore the Tablets
+	ver, err := getVTExecVersion("vttablet")
+	require.NoError(t, err)
+	if ver > 15 {
```

Comment:
Are you planning to backport this work? If not, I'm not sure why this is needed.

Reply:
This check is for downgrade/upgrade test